app/containers/HomeContainer/index.js

```diff
@@ -70,15 +79,26 @@ export function HomeContainer({
     if (rName) {
       dipatchGithubRepos(rName);
       setLoading(true);
+      setFlag(false);
     }
   };
+
+  const customHandleOnChange = rName => {
+    if (rName) {
+      setText(rName);
+      dipatchGithubRepos(rName);
+      setFlag(true);
```

Comment:
why do we need this flag here?

Reply:
If i am not use setFlag here then it's always false and return all repository, so only search feeling lucky